components/molecules/cards/card-project/card-project.types.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentPropsWithoutRef, ElementType, ReactNode } from "react";
+
+import { AvatarPort } from "components/atoms/avatar";
+import { ButtonPort } from "components/atoms/button/button.types";
+import { PaperPort } from "components/atoms/paper";
+import { TagPort } from "components/atoms/tag";
+
+interface Variants {}
+
+interface ClassNames {
+  base: string;
+}
+
+export interface CardProjectPort<C extends ElementType> extends Partial<Variants> {
+  as?: C;
+  paperProps?: Partial<PaperPort<C>>;
+  htmlProps?: ComponentPropsWithoutRef<C>;
+  classNames?: Partial<ClassNames>;
+  avatarProps: AvatarPort;
+  title?: ReactNode;
+  description?: ReactNode;
+  topTags?: Array<TagPort<"div">>;
+  bottomTags?: Array<TagPort<"div">>;
+  primaryAction?: ButtonPort<"button">;
+  secondaryAction?: ButtonPort<"button">;
```

Comment:
```suggestion
  titleProps?: ReactNode;
  description?: ReactNode;
  topTags?: Array<TagPort<"div">>;
  bottomTags?: Array<TagPort<"div">>;
  primaryActionProps?: ButtonPort<"button">;
  secondaryActionProps?: ButtonPort<"button">;
```
We have similar naming in other molecules, same issue in your other PR. We can discuss this.

Reply:
And shouldn't the title be required ?